imports/startup/server/tags.js

```diff
@@ -26,14 +26,16 @@ Meteor.methods({
       $pull: { taggedSongs: { tags: tagId}}
     })
   },
-
-
   addSongTag: (userId, tagId, songId) => {
     const songExists = Meteor.users.findOne({ "_id": userId, "taggedSongs.id": songId });
     if (songExists){
-        Meteor.users.update({ "_id": userId, "taggedSongs.id": songId }, {
-            $push: { "taggedSongs.$.tags": tagId }
-        });
+        // check to see if song is already tagged with the same tag
```

Comment:
I pushed a similar change as well. Just making note in case I have to refactor. Nothing wrong with it.